README.md

```diff
@@ -34,3 +34,5 @@ $ npm install
 ```
 $ npm run watch
 ```
+
+Note: this component requires `com.fliplet.interactive-map` to be the suffix of the widget.json `package` name, e.g. when making changes to it just add a suffix like `com.fliplet.interactive-map.foo-bar`.
```

Comment:
@squallstar I think you meant prefix here, because you've used `[data-widget-package^="com.fliplet.interactive-map"]` in the CSS selector, where the `^=` means a class attribute **starting with** `com.fliplet.interactive-map`.

Reply:
Yes I've changed it in a later commit ;)